### PR TITLE
Add confetti animation

### PR DESCRIPTION
**Pull request by riyuzenn**
### Description

Add confetti animation if the user reach new pb highscore and a daily leaderboard rank.

https://user-images.githubusercontent.com/66673988/192303067-37f4b0fd-59a3-41cd-a3ff-1bd5f50f79a5.mp4

Closes #

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "npm": "8.1.2"
   },
   "devDependencies": {
+    "@types/canvas-confetti": "^1.4.3",
```
Comment (Bruception) [inline frontend/package.json:20]: Save exact

File: frontend/package.json [hunk 2]
```diff
@@ -55,6 +56,7 @@
   },
   "dependencies": {
     "axios": "0.21.4",
+    "canvas-confetti": "^1.5.1",
```
Comment (Bruception) [inline frontend/package.json:59]: Save exact

File: frontend/src/ts/test/result.ts [hunk 3]
```diff
@@ -342,6 +344,37 @@ export function showCrown(): void {
...
+  const style = getComputedStyle(document.body);
+  const colors = [
+    style.getPropertyValue("--main-color"),
+    style.getPropertyValue("--text-color"),
+  ];
+  const duration = Date.now() + 2500;
+
+  (function f(): void {
+    confetti({
+      particleCount: 2,
```
Comment (Bruception) [inline frontend/src/ts/test/result.ts:358]: How does this look in mobile, other screen sizes?

**Comment by Miodec** [comment]
I like it but im thinking its doesn't really fit the style of the website..

**Comment by Bruception** [comment]
Looks cool. Will need approval from @Miodec in terms of product design

File: frontend/src/ts/test/result.ts (hunk 3 above)
Comment (riyuzenn) [reply frontend/src/ts/test/result.ts:358]: ## mobile
https://user-images.githubusercontent.com/66673988/192457402-18115c24-e345-4f14-a97e-72e76df99f66.mp4

## ipad / tablet
https://user-images.githubusercontent.com/66673988/192457475-db4c931c-c9d7-4e9b-875a-7eab3e66e12d.mp4

**Comment by rizwanmustafa** [comment]
@Miodec Looks cool to me. Plus it's kinda resembles the powermode. I think this would be a great feature.

**Pull request merged**